### PR TITLE
fix(get_any_ks_cf_list): Use blocking call to fetch result

### PR DESCRIPTION
**Pull request by aleksbykov**
during job longevity-cdc-3d-400gb-test, when nemesis get list of
all tables in cluster the session.execute_async call failed with
error but this error was not processed correctly and nemesis run
stucked. sct issue #4417.

Removed the PageFetcher and use block call async execution

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by aleksbykov** [comment]
Job with fix is passing snapshot operation: https://jenkins.scylladb.com/view/scylla-4.6/job/scylla-4.6/job/SCT-Reproducers/job/longevity-cdc-3d-400gb-test/2 and continue running

File: sdcm/cluster.py [hunk 1]
```diff
@@ -3353,13 +3352,16 @@ def execute_cmd(cql_session, entity_type):
             else:
                 raise ValueError(f"The following value '{entity_type}' not supported")
 
-            session.default_fetch_size = 1000
-            session.default_consistency_level = ConsistencyLevel.ONE
-            execute_result = session.execute_async(cmd)
-            fetcher = PageFetcher(execute_result).request_all()
-            current_rows = fetcher.all_data()
```
Comment (fruch) [inline sdcm/cluster.py:3360]: what is exactly the problem with `PageFetcher` ? 
in you code you are not doing pagination at all, and you might get partial data

File: sdcm/cluster.py [hunk 2]
```diff
@@ -3353,13 +3352,16 @@ def execute_cmd(cql_session, entity_type):
...
+            cql_session.default_fetch_size = 1000
+            cql_session.default_consistency_level = ConsistencyLevel.ONE
+            execute_result_future = session.execute_async(cmd, timeout=120)
```
Comment (roydahan) [inline sdcm/cluster.py:3357]: @aleksbykov what happens if we use it in places where we have more than 1000 results?
Or even 1000 results take too long to come back, and we need smaller pages.

(I don't know if we do it or not).

Isn't the original problem you're trying to solve is much simpler?
We just need to make sure the execute_async completed returned.

**Comment by aleksbykov** [comment]
PR is not relevant

**Pull request closed without merging**